common/views/components/Header/Header.js

```diff
@@ -1,17 +1,16 @@
 // @flow
-import { withToggler } from '../../hocs/withToggler';
+import { useState } from 'react';
```

Comment:
Love it! That `withToggler` was gold award over-engineering.

Reply:
We didn't have hooks at the time... without state management it was OK?

Reply:
I think even then on balance I'd have preferred the `setState` api. But there's no question that this is better now.